src/plugins/data/common/search/expressions/esql.ts

```diff
@@ -315,7 +315,14 @@ export const getEsqlFn = ({ getStartDependencies }: EsqlFnArguments) => {
             (body.all_columns ?? body.columns)?.map(({ name, type }) => ({
               id: name,
               name,
-              meta: { type: esFieldTypeToKibanaFieldType(type), esType: type },
+              meta: {
+                type: esFieldTypeToKibanaFieldType(type),
+                esType: type,
+                sourceParams: {
+                  appliedTimeRange: input?.timeRange,
+                  params: {},
+                },
```

Comment:
This actually is attached to every column, but it looks like it used only for the Date type columns. Can we just add that parameter only in that case?

Reply:
addressed in https://github.com/elastic/kibana/pull/201540/commits/46eb97e931190afc842655c7afec8ef972b44d4c